_docs-2/configuration/overview.md

```diff
@@ -86,13 +86,26 @@ client.tableOperations().setProperty("mytable", "table.durability", "log");
 
 ### Zookeeper Considerations
 
+#### Storing Accumulo Server Properties
+
 Any [server properties] that are set in Zookeeper should consider the limitations of Zookeeper itself with respect to the
 number of nodes and the size of the node data. Custom table properties and options for Iterators configured on tables
 are two areas in which there aren't any fail safes built into the API that can prevent the user from making this mistake.
 
 While these properties have the ability to add some much needed dynamic configuration tools, use cases which might fall
 into these warnings should be reconsidered.
 
+#### Monitor and Zookeeper Four-Letter-Word command whitelist
+
+Starting with Zookeeper version 3.5.3, the Four Letter Words commands must be enabled with the 
+zookeeper property: 4lw.commands.whitelist.  The monitor uses the stat command to provide zookeeper server(s),
+mode and number of clients.  At a minimum, add the following to your zookeeper configuration (zoo.cfg):
+```
+4lw.commands.whitelist=stat
+```
+See [Zookeeper Administration Documentation] for your zookeeper version for other four-letter-word 
+command whitelist options.
+
```

Comment:
I don't think this section is necessary at all. It should be sufficient to link directly to ZK docs on the matter, rather than duplicate the information here, so we don't become a configuration guide for ZK itself. See my other review comments for more.

```suggestion
```